From: Andrew Bower <andrew@bower.uk>
Date: Sun, 21 Apr 2024 20:23:41 +0100
Subject: update for autoconf obsolescence

---
 configure.ac | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 5a7900d..a3cbaca 100644
--- a/configure.ac
+++ b/configure.ac
@@ -26,13 +26,11 @@ AC_CONFIG_HEADERS([src/config.h])
 # Checks the compiler vendor
 AX_COMPILER_VENDOR
 AC_PROG_CC
-AC_PROG_CC_STDC
 
 # GPGME is compiled with _FILE_OFFSET_BITS=64 on i386
 AC_SYS_LARGEFILE
 
 # Checks for header files
-AC_HEADER_STDC
 AC_CHECK_HEADERS([stdlib.h string.h unistd.h])
 
 # Checks for functions and libraries
