From: Andrew Bower <andrew@bower.uk>
Date: Tue, 23 Apr 2024 19:34:45 +0100
Subject: Appease -Wunused-variable

---
 src/rc.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/rc.c b/src/rc.c
index 47c56b4..9211df8 100644
--- a/src/rc.c
+++ b/src/rc.c
@@ -72,7 +72,9 @@ read_rc(const char *file)
 	char *vals[2] = {0};           /* Key, value read from a line */
 	struct stat buf = {0};         /* Stat information */
 
+#ifdef HAVE_UNVEIL
 	static const char nfile[] = ".netrc";  /* Netrc file */
+#endif
 	static const char rfile[] = ".mcdsrc"; /* Default rc file */
 
 	if (file == NULL) {
