From: Andrew Bower <andrew@bower.uk>
Date: Sun, 21 Apr 2024 20:24:29 +0100
Subject: fix lintian-found typos

---
 src/decrypt.c | 2 +-
 src/mcds.1    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/decrypt.c b/src/decrypt.c
index f9e423a..ee2db24 100644
--- a/src/decrypt.c
+++ b/src/decrypt.c
@@ -82,7 +82,7 @@ decrypt(char *filename)
 
 	gpgme_check_version(NULL);
 	if ((err = gpgme_engine_check_version(GPGME_PROTOCOL_OpenPGP)) != 0) {
-		warnx(_("Unable to initalize GPGME: %s"), gpgme_strerror(err));
+		warnx(_("Unable to initialize GPGME: %s"), gpgme_strerror(err));
 		return(EXIT_FAILURE);
 	}
 
diff --git a/src/mcds.1 b/src/mcds.1
index 1f39299..c54b335 100644
--- a/src/mcds.1
+++ b/src/mcds.1
@@ -119,7 +119,7 @@ will not use the
 .Pa ~/.netrc
 file.
 .It Cm password_file No \&= Ar password.gpg
-The GPG encrypted file containg the password for the CardDAV server.
+The GPG encrypted file containing the password for the CardDAV server.
 .El
 .It Pa ~/.netrc
 Used to access your username and password when authenticating with the
