From: Andrew Bower <andrew@bower.uk>
Date: Tue, 23 Apr 2024 19:35:07 +0100
Subject: Fix -Wstringop-truncation

---
 src/rc.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/rc.c b/src/rc.c
index 9211df8..3736583 100644
--- a/src/rc.c
+++ b/src/rc.c
@@ -125,8 +125,9 @@ read_rc(const char *file)
 				vals[i] = xmalloc(len +1);
 				if (tmp[len-1] == '\n') {
 					strncpy(vals[i], tmp, len-1);
+					vals[i][len] = '\0';
 				} else {
-					strncpy(vals[i], tmp, len);
+					strncpy(vals[i], tmp, len+1);
 				}
 				++i;
 				if (i == 2) {
