From: Andrew Bower <andrew@bower.uk>
Date: Tue, 23 Apr 2024 23:09:16 +0100
Subject: Format size_t correctly on i386 (-Wformat)
Forwarded: https://github.com/t-brown/mcds/pull/18/commits/5766af33556bea98a1bf031be05aa0b27dbfdc59

---
 src/mem.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mem.c b/src/mem.c
index 9592f3f..0c1317b 100644
--- a/src/mem.c
+++ b/src/mem.c
@@ -57,7 +57,7 @@ xmalloc(size_t n)
     memset(ptr, 0, n);
     return ptr;
   } else {
-    errx(EX_SOFTWARE,_("out of memory (unable to allocate %ld bytes)"), n);
+    errx(EX_SOFTWARE,_("out of memory (unable to allocate %ld bytes)"), (long) n);
   }
   /* should never get here */
   return NULL;
